src/storage_tiering.cpp

```diff
@@ -979,14 +978,12 @@ namespace irods {
            const_cast<char*>(access_time.c_str()),
            const_cast<char*>(config_.migration_scheduled_flag.c_str())};
 
-        auto status = exec_as_user(_comm, _user_name, _user_zone, [&set_op](auto comm) -> int {
-                            return rcModAVUMetadata(comm, &set_op);
-                            });
-        if(status < 0) {
-           THROW(
-               status,
-               boost::format("failed to set migration scheduled flag for [%s]")
-               % _object_path);
+        if (_comm->clientUser.authInfo.authFlag >= LOCAL_PRIV_USER_AUTH) {
+            addKeyVal(&set_op.condInput, ADMIN_KW, "true");
```

Comment:
Let's replace `"true"` with an empty string to avoid confusing ourselves and new readers of the code.